netbox/core/tables/plugins.py

```diff
@@ -56,11 +56,12 @@ class CatalogPluginTable(BaseTable):
     is_certified = columns.BooleanColumn(
         verbose_name=_('Certified')
     )
-    created_at = columns.DateTimeColumn(
-        verbose_name=_('Published')
+    installed_version = tables.Column(
+        verbose_name=_('Installed Version')
     )
-    updated_at = columns.DateTimeColumn(
-        verbose_name=_('Updated')
+    latest_version = tables.Column(
+        accessor=tables.A('release_latest__version'),
+        verbose_name=_('Latest Version')
     )
```

Comment:
We should keep these columns as optional, as they still convey data from the catalog API.